app/jobs/resolution_proofing_job.rb

```diff
@@ -39,22 +48,48 @@ def perform(result_id:, encrypted_arguments:, trace_id:, should_proof_state_id:,
                           )
                         end
 
+    add_threatmetrix_result_to_callback_result(callback_log_data.result, threatmetrix_result)
```

Comment:
This is missing a critical guard, fixed in https://github.com/18F/identity-idp/pull/6765